vtex-apps/order-items
# CHK-464: Update enqueue to aggregate orderForm messages

#### What problem is this solving?

Aggregates the orderForm messages of intermediate tasks, so we call show the user all the messages related to the items they have changes. E.g.: if a user updates an item quantity to above the maximum quantity of SKUs the store has configured, and quickly adds another product to the cart which changes the selected shipping option, they would only see the messages regarding this last addition, and would not be aware that they've updated the quantity of the first item to above the store's limit.

#### How to test it?

[Workspace](https://toasts--checkoutio.myvtex.com/).

Add the first product on the shelf and update its quantity to 51, then add the second product on the shelf. You should see both messages like I described above.

#### Screenshots or example usage:

#### Describe alternatives you've considered, if any.

This logic shouldn't be here. We need to make an effort and move this and the `if (queueStatusRef.current === QueueStatus.FULFILLED) { setOrderForm(...) }` logic into `order-manager/OrderQueue` or `order-manager/OrderForm` where we would benefit from this in every other orchestrator without any code changes.

#### Related to / Depends on

N/A
